packages/valory/skills/abstract_round_abci/behaviour_utils.py

```diff
@@ -95,6 +95,8 @@
 HEIGHT_OFFSET_MULTIPLIER = 0.01
 NON_200_RETURN_CODE_DURING_RESET_THRESHOLD = 3
 GENESIS_TIME_FMT = "%Y-%m-%dT%H:%M:%S.%fZ"
+ROOT_HASH = "726F6F743A3"
+RESET_HASH = "72657365743A3"
```

Comment:
The root hash is of the form:
`726F6F743A3{ROUND_COUNT}72657365743A3{RESET_INDEX}`
which is the hash that Tendermint produces from:
`root:{ROUND_COUNT}reset:{RESET_INDEX}`